openedx/features/course_experience/waffle.py

```diff
@@ -0,0 +1,17 @@
+"""
+Miscellaneous waffle switches that both LMS and Studio need to access
+"""
+from openedx.core.djangoapps.waffle_utils import WaffleSwitchNamespace
+
+# Namespace
+WAFFLE_NAMESPACE = u'course_experience'
+
+# Switches
+ENABLE_COURSE_ABOUT_SIDEBAR_HTML = u'enable_about_sidebar_html'
```

Comment:
This seems like a good place for waffle definitions for the course_experience feature...